### ISSUE TITLE
Edit tools conf XML

### ISSUE THREAD

**Issue by pierrickrogermele** [workflow4metabolomics/w4m-vm#22]
Add installed tools inside XML config file.

**Comment by lecorguille** [comment]
What do you mean?

**Comment by pierrickrogermele** [comment]
I'm currently trying to install tools from github sources.
I've begun with lcmsmatching, and now I need to edit the XML config file automatically in order to add the tool.

**Comment by pierrickrogermele** [comment]
Done

**Comment by lecorguille** [comment]
Can you attach a link to your PR or commit, please?
It's really hard to follow

**Comment by pierrickrogermele** [comment]
https://github.com/workflow4metabolomics/w4m-vm/commit/b3c4cff120a82fc36b15b627759826f61c172d9a
